### ISSUE TITLE
Eliminate dependency on deprecated 'sass' Gem

### ISSUE THREAD

**Issue by jdickey** [rtomayko/tilt#334]
When doing a `bundle install` of any app requiring the `sass` Gem, one gets the following message after completion:

```
Post-install message from sass:

Ruby Sass is deprecated and will be unmaintained as of 26 March 2019.

* If you use Sass as a command-line tool, we recommend using Dart Sass, the new
  primary implementation: https://sass-lang.com/install

* If you use Sass as a plug-in for a Ruby web framework, we recommend using the
  sassc gem: https://github.com/sass/sassc-ruby#readme

* For more details, please refer to the Sass blog:
  http://sass.logdown.com/posts/7081811
```

A cursory inspection suggests that the only changes necessary for Tilt to support [`sassc`](https://github.com/sass/sassc-ruby) 2.0.0 would be to change `lib/tilt/sass.rb` to `require 'sassc'` rather than ['sass'](https://github.com/rtomayko/tilt/blob/master/lib/tilt/sass.rb#L2) and to change the `#prepare` method to instantiate `::SassC::Engine` rather than the current [`::Sass::Engine`](https://github.com/rtomayko/tilt/blob/master/lib/tilt/sass.rb#L13).

**Comment by krisleech** [comment]
I can confirm this branch fixes an issue with the haml gem where inline sass fails with `"scss" filter's sass dependency missing: try installing it or adding it to your Gemfile`.

`gem 'tilt', git: 'https://github.com/jdickey/tilt.git', branch: 'hotfix/334-replace-sass-with-sassc'`

**Comment by judofyr** [comment]
I hear you. I stumbled upon this myself. Let me see if I can have a look at it this week.

**Comment by judofyr** [comment]
Build for 2.0.9 running now: https://travis-ci.com/judofyr/tilt/builds/92808753 (Not sure why, but the rtomayko Travis CI repo isn't running)

**Comment by zbynek** [comment]
I guess this issue can be closed  (FTR fixed in https://github.com/rtomayko/tilt/commit/9feb9168382e4ee80070db0f5919e3c78bb923a5 )

**Comment by jdickey** [comment]
Closing; appears fixed in 2.0.9. Thanks, @judofyr and all. Sorry for the atypically slow response; I’ve spent the last month in and out of hospital. Health improved; bank balance, less so. 😩
